commands/create_command.py

```diff
@@ -0,0 +1,26 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema):
+        self._schema = schema
+
+    def execute(self):
+        if (self._schema == None or not os.path.exists(parent_dir + "/" + self._schema)):
+            raise Exception("FileNotFound")
+
+        schema_file = open(self._schema, "r")
+        data = json.load(schema_file)
```

Comment:
I would suggest converting the schema file to the data in the constructor. Execute should deal with schema as an object.